elastic/kibana
# [Reporting] Wholesale moves client to newest-platform

## Summary

Moves all of our UI-facing architecture to newest platform.

## TODO

- [x] Remove client code in legacy space
- [x] Remove injectedVars and other artifacts in the legacy server
- [x] Update Panel Actions to be new-platform
- [x] Fix and update any tests
- [x] Final smoke test (license changes, rejection messages, and any other regressions).

## QA List

- Biggest thing to check is license handling since all logic got ported to the client.
- Want to ensure that all the prior functionality was there:
  - Gold licenses should have PNG/PDF reporting working.
  - Basic licenses should have just CSV exporting from Discover and Saved Searches.
  - Open should have none of the above options.
